react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export default class CustomButton extends React.Component {
+  constructor(prop) {
+    super(prop);
+    this.state = {
+      className: 'none',
+      buttonCounter: 0
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  render() {
+    if (this.state.buttonCounter >= 18) {
+      return <button onClick={this.handleClick} className={this.state.className}>Hot Button</button>;
```

Comment:
here you would pass `className` as `className={this.hotButton()}` instead of `this.state.className`